src/context/auth.tsx

```diff
@@ -41,7 +43,7 @@ export const AuthProvider: FC<ProviderProps> = ({ children }) => {
 
   const { data: projects = [], isLoading: isProjectsLoading } = useQuery({
     queryKey: [queryKeys.projects],
-    queryFn: fetchProjects,
+    queryFn: () => fetchProjects(false),
```

Comment:
This seems still wrong, but I think after merging the images PR and rebasing this goes away anyhow.